recipe/cross_compile_support.sh

```diff
@@ -108,6 +109,9 @@ if [[ "${HOST_PLATFORM}" != "${BUILD_PLATFORM}" ]]; then
                 mkdir -p /opt/conda/targets/
                 mv ./usr/local/cuda-${CUDA_COMPILER_VERSION}/targets/${CUDA_HOST_PLATFORM_ARCH}-linux /opt/conda/targets/${CUDA_HOST_PLATFORM_ARCH}-linux
                 /usr/bin/sudo cp -r /opt/conda/targets/${CUDA_HOST_PLATFORM_ARCH}-linux ${CUDA_HOME}/targets/${CUDA_HOST_PLATFORM_ARCH}-linux
+
+		# Need libcuda.so.1 in test phase to test the executables
+		mv ./usr/local/cuda-${CUDA_COMPILER_VERSION}/compat/* ${QEMU_LD_PREFIX}/usr/lib/
```

Comment:
bash may not care but it looks like this block needs to be indented maybe?

Reply:
I'm not sure I understand. They have the same indentation as Line 110 right?

Reply:
I don't think so.

<img width="937" alt="Screenshot 2023-04-18 at 10 45 10 AM" src="https://user-images.githubusercontent.com/5296416/232831352-0a5ddbb6-69dc-4e2b-83e3-17fe7a14e743.png">

Reply:
Are there tabs not being rendered correctly?

Reply:
This is what I see

![image](https://user-images.githubusercontent.com/5234427/232831788-4a89ac04-93d1-46d4-ad15-6122001a8e27.png)

Reply:
OMG.